.ci/build_packages/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG BUILD_FROM=emqx/build-env:erl22.3-ubuntu20.04
+FROM ${BUILD_FROM}
+
+ARG EMQX_NAME=emqx
+
+COPY . /emqx
+
+WORKDIR /emqx
+
+RUN sed -i "/^export LC_ALL=.*$/d" Makefile && make ${EMQX_NAME}-pkg || cat rebar3.crashdump
```

Comment:
Question: we don't have `LC_ALL` defined in Makefile.
what was the reason we needed it in emqx-rel project, but then delete the line from Dockerfile?

Reply:
The reason we need it in the rel project is because if the user's git command outputs Chinese, then rebar.config.script will report an error
Now i will delete it